public/content/sv/welcome/install-d-locally.md

```diff
@@ -0,0 +1,43 @@
+# Installera D lokalt
+
+På D språkets webbsida [dlang.org](https://dlang.org) kan den senaste kompilerar
+versionen av referens kompilatorn **DMD** (Digital Mars D)
+[laddas ned](http://dlang.org/download.html) och installeras:
+
+### Windows
+
+* [Installerare](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd-{{latest-release}}.exe)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.windows.7z)
+* genom [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* `.dmg` [fil](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.dmg)
```

Comment:
I know that this if what the English version says, but the DMG file contains and installer that is as much an installer as the Windows installer.